crates/burn-import/src/onnx/dim_inference.rs

```diff
@@ -266,6 +267,41 @@ fn reduce_mean_update_outputs(node: &mut Node) {
     }
 }
 
+/// Update the output tensor dimension
+fn squeeze_update_output(node: &mut Node) {
+    let axes = if node.inputs.len() == 2 {
+        match &node.inputs[1].value {
+            Some(value) => match value {
+                Data::Int64s(axes) => Some(axes.clone()),
+                _ => panic!("Squeeze: invalid input types"),
+            },
+            None => None,
+        }
+    } else {
+        node.attrs.get("axes").cloned().map(|v| v.into_i64s())
+    };
```

Comment:
LGTM! 🙂 

We lost the check to make sure a single axis is specified (since we don't support multiple axes in this PR).

Could you add that back? Possible just below with the `axes.is_none()` condition, e.g. :

```rust
if axes.is_none() {
    panic!("Squeeze must specify an axis");
} else if axes.len() > 1 {
    // some panic message
}

Reply:
Done!